resources/assets/js/components/App.vue

```diff
@@ -2,14 +2,14 @@
   <div>
     <facet-navigation></facet-navigation>
 
-    <section class="hero is-light is-bold">
+    <!-- <section class="hero is-light is-bold">
```

Comment:
feel free to delete if this section is replaced, where does the section tag end at this point?